cli/ceph/mgr.py

```diff
@@ -24,3 +24,12 @@ def module(self, action, module=None, force=False):
             cmd += " --force"
 
         return self.execute(sudo=True, long_running=True, cmd=cmd)
+
+    def fail(self, mgr_name):
+        """
+        Fail/down a given mgr
+        Args:
+            mgr_name (str): mgr to bring down
+        """
+        cmd = f"{self.base_cmd} fail {mgr_name}"
+        return self.execute(sudo=True, long_running=True, cmd=cmd)
```

Comment:
Add support for error check "check_ec"

https://github.com/red-hat-storage/cephci/blob/master/cli/ceph/balancer.py#L15

Reply:
You mean to add check_ec=True here?